docs/pull_request_template.md

```diff
@@ -7,3 +7,4 @@ Maintainers will complete the following section:
 - [ ] JSON/YAML configuration changes are updated in the relevant schema
 - [ ] Changes to metadata also update the documentation for the metadata
 - [ ] Pull request includes link to an osbs-docs PR for user documentation updates
+- [ ] New feature can disabled from configuration file
```

Comment:
How about "New feature can be disabled from configuration file"?

Reply:
fixed, thanks